### DISCUSSION TITLE
[ 71%] Built target aws_sdk_s3_ep, No response for a long time

### DISCUSSION THREAD

**Discussion by Leo9866** [milvus-io/milvus#24994] [Q&A and General discussion]
**When I execute the make command, I start compiling, and when I get to Built target aws_sdk_s3_ep, there is no task response. All the way to the error message below**

**My environment is ubuntu 20.04, 16C32G, and another 24C48G is the same problem**

make[4]: Leaving directory '/mnt/milvus/cmake_build/3rdparty_download/aws-sdk-subbuild/src/aws_sdk_s3_ep'
[ 71%] Completed 'aws_sdk_s3_ep'
make[3]: Leaving directory '/mnt/milvus/cmake_build'
[ 71%] Built target aws_sdk_s3_ep
CMake Error at /mnt/milvus/cmake_build/3rdparty_download/arrow-subbuild/src/arrow-ep-stamp/arrow-ep-build-Release.cmake:49 (message):
  Command failed: 2

   'make'

  See also

    /mnt/milvus/cmake_build/3rdparty_download/arrow-subbuild/src/arrow-ep-stamp/arrow-ep-build-*.log

thirdparty/arrow/CMakeFiles/arrow-ep.dir/build.make:130: recipe for target '3rdparty_download/arrow-subbuild/src/arrow-ep-stamp/arrow-ep-build' failed
make[3]: *** [3rdparty_download/arrow-subbuild/src/arrow-ep-stamp/arrow-ep-build] Error 1
make[3]: Leaving directory '/mnt/milvus/cmake_build'
CMakeFiles/Makefile2:1171: recipe for target 'thirdparty/arrow/CMakeFiles/arrow-ep.dir/all' failed
make[2]: *** [thirdparty/arrow/CMakeFiles/arrow-ep.dir/all] Error 2
make[2]: Leaving directory '/mnt/milvus/cmake_build'
Makefile:170: recipe for target 'all' failed
make[1]: *** [all] Error 2
make[1]: Leaving directory '/mnt/milvus/cmake_build'
Makefile:139: recipe for target 'build-cpp' failed
make: *** [build-cpp] Error 1

**Comment by yhmo** [comment]
I guess it failed to download arrow. You can check the err log in the folder:

![Screenshot from 2023-06-19 19-09-43](https://github.com/milvus-io/milvus/assets/2282099/49bcb805-1bc8-4ce6-9a41-e1212b1c9718)
